README.md

```diff
@@ -152,3 +157,18 @@ Add user to the list of online players, returns the list.
       "bob"
     ]
 
+# Push Notifications API
+
+## Save Push Token [POST /<auth>/push-token]
+
+Saves user's push notifications token to database. Example Body:
+
+``` js
+{ username: 'alice',             // String, which user this token is for
+  app: 'substract-game',         // String, which app this token is for
+  type: 'ios',                   // String, which OS this token is for,
```

Comment:
Well, if this matters, I think type will be either `apn` (apple push notifications) or `gcm` (google cloud messaging), which are the usual acronyms.

For Android there are other push notifications providers than Google. For iOS, maybe there is, I don't know...